src/components/SecurityPage.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react'
+import Button, { ButtonLink } from 'cozy-ui/transpiled/react/Button'
+import AppLinker, { generateWebLink } from 'cozy-ui/transpiled/react/AppLinker'
+import { translate } from 'cozy-ui/transpiled/react/I18n'
+import { withClient } from 'cozy-client'
+import compose from 'lodash/flowRight'
+import { Link } from 'react-router-dom'
+import Wrapper from 'components/Wrapper'
+import NarrowContent from 'cozy-ui/transpiled/react/NarrowContent'
+import strongPasswordIcon from 'assets/strong-password.svg'
+import Stack from 'cozy-ui/transpiled/react/Stack'
+import { MainTitle, Text } from 'cozy-ui/transpiled/react/Text'
+import Card from 'cozy-ui/transpiled/react/Card'
+import { UnorderedList, ListItem } from 'cozy-ui/transpiled/react/UnorderedList'
+import snarkdown from 'snarkdown'
+
+const DumbLinkToSettings = withClient(props => {
+  // eslint-disable-next-line no-unused-vars
```

Comment:
By curiosity, why do we need this eslint pragma?

Reply:
Because `f` is not really used but only destructured so it's not part of `rest`